components/conversations/ThreadActivity.js

```diff
@@ -207,14 +232,17 @@ const ThreadActivity = ({ activity }) => {
       {message && (
         <ActivityParagraph activityColor={activityColor} mt={1} fontSize="12px" whiteSpace="pre-line">
           <ActivityMessage color={activityColor}>
-            {formatMessage(message, { movedFromCollective: activity.data?.movedFromCollective?.name || 'collective' })}
+            {intl.formatMessage(message, {
+              movedFromCollective: activity.data?.movedFromCollective?.name || 'collective',
+            })}
           </ActivityMessage>
           {details && (
             <Fragment>
               <br />
               {details}
             </Fragment>
           )}
+          {DataRenderer && <DataRenderer activity={activity} />}
```

Comment:
Reusing details would be simpler, no?
```
details = renderDetailsString({
        ...activity.transaction,
        isCredit: activity.transaction.type === 'CREDIT',
        intl,
      })
```

Reply:
I initially started with that, but even though it's not used by "Paid" activity yet I didn't want to override the existing `details` thing which is a different concept (un-typed details from the API); both should be able to co-exist.